### PR TITLE
i18n(zh-cn): Update `buttercms.mdx` translation

### PR DESCRIPTION
**Pull request by hexWars**
#### Description (required)

Modify some translations.

#### Related issues & labels (optional)

- Closes #
- Suggested label: i18n

### CODE REVIEW

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 1]
```diff
@@ -21,7 +21,7 @@ ButterCMS 是一种允许你在项目中发布结构化内容的 headless CMS和
 
 1. **Astro 项目** -如果你还没有 Astro 项目，我们的[安装指南](/zh-cn/install/auto/)将帮助你快速创建并运行一个项目。
 
-2. **ButterCMS帐户**. 如果你还没有ButterCMS帐户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
+2. **ButterCMS账户**. 如果你还没有ButterCMS账户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:24]:
```suggestion
2. **ButterCMS 账户**. 如果你还没有 ButterCMS 账户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
```
Looks great. We can add a space by the way

File: src/content/docs/zh-cn/guides/cms/buttercms.mdx [hunk 2]
```diff
@@ -21,7 +21,7 @@ ButterCMS 是一种允许你在项目中发布结构化内容的 headless CMS和
 
 1. **Astro 项目** -如果你还没有 Astro 项目，我们的[安装指南](/zh-cn/install/auto/)将帮助你快速创建并运行一个项目。
 
-2. **ButterCMS帐户**. 如果你还没有ButterCMS帐户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
+2. **ButterCMS账户**. 如果你还没有 ButterCMS 账户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
```
Comment (liruifengv) [inline src/content/docs/zh-cn/guides/cms/buttercms.mdx:24]:
```suggestion
2. **ButterCMS 账户**. 如果你还没有 ButterCMS 账户，你可以通过[注册](https://buttercms.com/join/) 免费试用来创建一个。
```

Comment (hexWars) [reply src/content/docs/zh-cn/guides/cms/buttercms.mdx]: great

**Review by liruifengv — APPROVED** [review]
LGTM! Thanks fix this typo. @hexWars

**Pull request merged**